pkg/api/v1/routes/handlers.go

```diff
@@ -227,20 +240,26 @@ func (r *Routes) serverEnroll(c *gin.Context) (int, *v1types.ServerResponse) {
 	// Delete params related to add servers.
 	var inventoryArgs rctypes.InventoryTaskParameters
```

Comment:
We shouldn't be accepting this from consumers of the API, we should just dictate it as `outofband`, `collect_bios_cfg = true`, `collect_firmware_true`. We already have `asset_id` too. That save a lot of boilerplate concerns over cleaning up on JSON marshaling that in practice will never be a concern.

@joelrebel would you agree that these are the correct inventory parameters to pass here?

Reply:
Do you suggest we hard code the parameters for creating the new condition?

This means we only need to set `asset_id` to `serverID` where other parameters will be hard coded as `outofband`, `collect_bios_cfg = true`, `collect_firmware_true`?

Reply:
Yes. I think clients of `serverEnroll` don't need to know (and shouldn't care) that we do an inventory as part of enrollment. It also insulates them from any changes we make later either in how we perform an inventory, or if we add additional steps to the enrollment process.